tools/github/print-runtime-release-issue.ts

```diff
@@ -32,30 +32,39 @@ async function main() {
     `
 ## Release
 - [ ] Check all proxy types.
-- [ ] Re-run all extrinsics/hooks benchmarks.
-- [ ] Branch from master and create branch \`perm-runtime-${newVersion}\`.
-- [ ] Tag \`perm-${newVersion}\` with runtime-${newVersion} and push to github
-- [ ] Tag \`perm-runtime-${newVersion}\` with runtime-${newVersion}-templates and push to github
-- [ ] Tag \`perm-runtime-${newVersion}\` with runtime-${newVersion}-starlight and push to github
-- [ ] NOTE: if this is a hotfix to one of the runtimes, branch from runtime-${previousVersion} version
-and create perm-runtime-${newVersion}-templates, perm-runtime-${newVersion}-starlight or perm-runtime-${newVersion}-tanssi
-depending on whether the hotfix is for templates, starlight or tanssi-parachain. Then tag accordingly
-- [ ] Start the github action Publish Runtime Draft
-with runtime-${previousVersion} => runtime-${newVersion}
-  - \`gh workflow run "Publish Runtime Draft" -r 'master' ` +
-    `-f from=runtime-${previousVersion} -f to=runtime-${newVersion} -f chains=tanssi-only\`
+
+### Tanssi-para
+- [ ] Branch from master and create branch \`perm-runtime-${newVersion}-para\`
```

Comment:
Just an idea, maybe we want to combine this to the single template, because it is the same instructions, but for 3 different types of networks, like:
```
const networkParams = [
   {"tagPostfix": "para", "networkName": "stagebox", "template": "orchestrator-para-only"},
   {...},
   {...},
]
for (const network of networkParams) {

}
```

<img width="858" alt="image" src="https://github.com/user-attachments/assets/04ab90cd-1d77-4a59-acad-ac37045c94b2" />

Reply:
yeah I am not that good with tyepescript and actions

Reply:
maybe let's leave that for another PR?

Reply:
and it's not that easy in the end, so I would nto bother that much